catalog/models.py

```diff
@@ -1,3 +1,94 @@
 from django.db import models
+from django.urls import reverse  # Used to generate URLs by reversing the URL patterns
+import uuid  # Required for unique book instances
 
-# Create your models here.
+
+class Genre(models.Model):
+    """Model representing a book genre."""
+
+    name = models.CharField(
+        max_length=200, help_text='Enter a book genre (e.g.Science Fiction)'
```

Comment:
I18n nội dung của option `help_text` em nhé, mục đích phục vụ cho tính năng đa ngôn ngữ sau này
https://docs.djangoproject.com/en/4.2/topics/i18n/translation/#internationalization-in-python-code

Sửa tương tự bên dưới